src/android/Utilities.java

```diff
@@ -23,6 +23,8 @@
  * Utilities class used for file and other common native operations.
  */
 public class Utilities {
+    private static final String ASSETS_MANIFEST_FILE = "cdvasset.manifest";
```

Comment:
`"cdvasset.manifest"` is actually filename, not the file itself, please use `ASSETS_MANIFEST_FILENAME` instead.